addons/hr_timesheet/models/project_task.py

```diff
@@ -182,6 +184,26 @@ def _get_timesheet(self):
         # Is override in sale_timesheet
         return self.timesheet_ids
 
+    def _get_timesheets_per_task(self):
+        tasks = self
+        timesheets_read_group = self.env['account.analytic.line']._read_group(
+            [('task_id', 'in', tasks.ids)],
+            ['task_id'],
+            ['id:recordset'],
+        )
+        return dict(timesheets_read_group)
```

Comment:
Actually, I will remove that method and move its code inside `_get_timesheet_report_data` since it is only used there. 🙂